mmv1/products/firebase/api.yaml

```diff
@@ -152,3 +152,61 @@ objects:
           Immutable. The globally unique, Firebase-assigned identifier of the App.
 
           This identifier should be treated as an opaque token, as the data format is not specified.
+  - !ruby/object:Api::Resource
+    name: 'AndroidApp'
+    min_version: beta
+    base_url: projects/{{project}}/androidApps
+    self_link: '{{name}}'
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      A Google Cloud Firebase Android application instance
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://firebase.google.com/'
```

Comment:
lets use [Add Firebase to your Android Project](https://firebase.google.com/docs/android/setup) instead

Reply:
Done.